action.yml

```diff
@@ -525,21 +532,21 @@ runs:
           ${{ env.CA_GH_ACTION_CONTAINER_APP_ENVIRONMENT_VARIABLES_ARG }}
 
     - name: Reset the ingress argument environment variable if it wasn't provided (use default ingress value)
-      if: ${{ env.CA_GH_ACTION_YAML_PROVIDED != 'true' && inputs.ingress == '' && env.CA_GH_ACTION_RESOURCE_EXISTS == 'true' }}
+      if: ${{ inputs.ingress == '' && env.CA_GH_ACTION_YAML_PROVIDED != 'true' && env.CA_GH_ACTION_RESOURCE_EXISTS == 'true' }}
       shell: bash
       run: |
         CA_GH_ACTION_INGRESS_ARG=""
         echo "CA_GH_ACTION_INGRESS_ARG=${CA_GH_ACTION_INGRESS_ARG}" >> $GITHUB_ENV
 
     - name: Reset the environment variables argument environment variable for the 'up' command
-      if: ${{ env.CA_GH_ACTION_YAML_PROVIDED != 'true' && inputs.environmentVariables != '' }}
+      if: ${{ env.CA_GH_ACTION_USE_UP == 'true' && inputs.environmentVariables != '' }}
       shell: bash
       run: |
         CA_GH_ACTION_CONTAINER_APP_ENVIRONMENT_VARIABLES_ARG="--env-vars ${{ inputs.environmentVariables }}"
         echo "CA_GH_ACTION_CONTAINER_APP_ENVIRONMENT_VARIABLES_ARG=${CA_GH_ACTION_CONTAINER_APP_ENVIRONMENT_VARIABLES_ARG}" >> $GITHUB_ENV
 
     - name: Update the existing Container App from provided arguments via 'up' (ingress values provided)
-      if: ${{ env.CA_GH_ACTION_YAML_PROVIDED != 'true' && (inputs.targetPort != '' || inputs.ingress != '') && env.CA_GH_ACTION_RESOURCE_EXISTS == 'true' }}
+      if: ${{ env.CA_GH_ACTION_USE_UP == 'true' && env.CA_GH_ACTION_YAML_PROVIDED != 'true' && env.CA_GH_ACTION_RESOURCE_EXISTS == 'true' }}
```

Comment:
Nit: I could be wrong here, but I'm not sure if it is necessary to check for `CA_GH_ACTION_YAML_PROVIDED` and `CA_GH_ACTION_RESOURCE_EXISTS` here since `USE_UP` can only be true if both of the prior env vars are true.

Reply:
Yep, that's a good point -- we can consolidate this conditional here